codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/OperationCustomization.kt

```diff
@@ -112,6 +113,19 @@ sealed class OperationSection(name: String) : Section(name) {
             writer.rustTemplate(".with_retry_classifier(#{classifier})", "classifier" to classifier)
         }
     }
+
+    data class FeatureGatedComponents(
```

Comment:
Looks like this can be removed now, unless you feel strongly that we'll need it again in the future.